src/legacy/core_plugins/kibana/public/home/tutorial_resources/logos/ibmmq.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="1000" height="400"><clipPath id="b"><path d="M0-1v83h55.937v236H1.523v82h194.65v-82h-55.938V82h54.414V-1zm554.02 1v82h55.938v236h-54.414v82h138.71V171.84l82.656 228.08 1.68.04 81.172-228.12V400h140.23v-82h-55.938V82h54.415V0h-156.33l-64.726 182.89L711.866 0z"/><path id="a" d="M222.23 0v82h55.937v130h215.31v-24.928s18.011-14.042 23.75-27.367l11.523-25.855s5.352-14.472 5.352-27.405l-2.305-25.855s-3.616-21.063-9.922-27.328l-22.266-25.895S471.276 0 433.75 0zm140.23 82h81.992v76H362.46z"/><use transform="matrix(1 0 0 -1 0 400)" xlink:href="#a"/></clipPath><path fill="none" stroke="#1f70c1" stroke-width="27.37" d="M0 13.683h1030v53.232H0m0 53.232h975v53.242H0v53.222h975v53.222H0m0 53.242h1030v53.242H0" clip-path="url(#b)"/></svg>
\ No newline at end of file
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 32 32"><defs><style>.cls-1,.cls-6{fill:none;}.cls-1{clip-rule:evenodd;}.cls-2{fill:#fff;}.cls-2,.cls-5,.cls-6{stroke:#7863ec;stroke-miterlimit:10;}.cls-3{clip-path:url(#clip-path);}.cls-4{fill:#e9e8ff;}.cls-5{fill:#e9e8fe;}</style><clipPath id="clip-path"><path class="cls-1" d="M10.55,16h13a1.5,1.5,0,0,1,0,3h-13a1.5,1.5,0,0,1,0-3Z"/></clipPath></defs><title>Artboard 1white</title><path class="cls-2" d="M10.59,8.51h11a1.5,1.5,0,1,1,0,3h-11a1.5,1.5,0,0,1,0-3Z"/><g class="cls-3"><rect class="cls-4" x="4.05" y="11" width="26" height="13"/></g><path class="cls-5" d="M9.51,14.52h13a1.5,1.5,0,1,1,0,3h-13a1.5,1.5,0,0,1,0-3Z"/><path class="cls-5" d="M10.6,20.51h11a1.5,1.5,0,0,1,0,3h-11a1.5,1.5,0,0,1,0-3Z"/><path class="cls-6" d="M2,4.55H3.73A1.73,1.73,0,0,1,5.45,6.29V24a3.52,3.52,0,0,0,3.5,3.53H23A3.52,3.52,0,0,0,26.51,24V6.31a1.75,1.75,0,0,1,1.74-1.76H30"/></svg>
```

Comment:
Could be simplified by stripping out debug information using an svg optimizer.

Reply:
Thanks, optimized.